src/plugins/intel_npu/tests/functional/behavior/infer_request_run.hpp

```diff
@@ -766,6 +767,22 @@ TEST_P(BatchingRunSeqTests, CheckMultipleBatchingRunsSeq) {
     }
 }
 
+using ROITensorInference = OVInferRequestInferenceTests;
+
+TEST_P(ROITensorInference, InferenceROITensor) {
+    auto shape_size = ov::shape_size(m_param.m_shape);
+    auto model = OVInferRequestInferenceTests::create_n_inputs(1, ov::element::f32, m_param.m_shape);
+    auto execNet = ie->compile_model(model, target_device);
```

Comment:
I think we should start naming `compiledModel` vars as `executable networks` are from the older API.

Reply:
thx for comment, done

Reply:
Also please consider using the camel case naming for the newly changed `auto compiled_model` variable, I think it sticks to our convention.

Reply:
where is this convention written or used? all tests from this file are using the snake method.